tests/DifferTest.php

```diff
@@ -323,6 +323,19 @@ public function textProvider(): array
                 "a\nb\nc\nd\ne\nf\ng\nh\ni\nj\nk",
                 "a\np\nc\nd\ne\nf\ng\nh\ni\nw\nk",
             ],
+            [
+                <<<EOF
+--- Original
++++ New
+@@ @@
+-A
++B
+
+EOF
+            ,
+                "A\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1",
+                "B\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1",
```

Comment:
this new case proves the common trailing chunk detection and skipping by the code that generates the diff string works (i.e. on the current master branch this will fail)